The Annotated Transformer.md

```diff
@@ -616,15 +632,37 @@ def get_std_opt(model):
 > Example of the curves of this model for different model sizes and for optimization hyperparameters. 
 
 ```python tags=[]
+## Enable altair to handle more than 5000 rows
+alt.data_transformers.disable_max_rows()
+
 # Three settings of the lrate hyperparameters.
 opts = [NoamOpt(512, 1, 4000, None), 
         NoamOpt(512, 1, 8000, None),
         NoamOpt(256, 1, 4000, None)]
-plt.plot(torch.arange(1, 20000), [[opt.rate(i) for opt in opts] for i in range(1, 20000)])
-plt.legend(["512:4000", "512:8000", "256:4000"])
-plt.gca().set_xlabel("Step")
-plt.gca().set_ylabel("Learning Rate")
-None
+
+
+opts = [NoamOpt(512, 1, 4000, None), 
```

Comment:
may ask you for a follow-up merge fix after optimizer changes https://github.com/austinvhuang/annotated-transformer/pull/4